### DISCUSSION TITLE
Quarkus 2.7.1.Final released - Maintenance release

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#23538] [Quarkus Blog/Website]
# Quarkus 2.7.1.Final released - Maintenance release

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-2-7-1-final-released/

**Comment by doogle-oss** [comment]
Thanks for the awsome work. Looking forward to oidc reactive.
